Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+@Library('podTemplateLib')
+import net.santiment.utils.podTemplates
+
+properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '30', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: ''))])
+
+slaveTemplates = new podTemplates()
+
+slaveTemplates.dockerTemplate { label ->
+  node(label) {
+    stage('Build') {
+      container('docker') {
+        def scmVars = checkout scm
+
+        if (scmVars.GIT_TAG_NAME.contains('relay')) {
```

Comment:
Why checking if tag contains `relay` ? Lets check for master branch: `if (env.BRANCH_NAME == 'master') { ...`

Reply:
Because changes can without relation to relay package. I added check for `main` branch and relay tag